queries.sql

```diff
@@ -6,3 +6,25 @@ SELECT (name,escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered is true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+BEGIN; UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+DELETE FROM animals;
+ROLLBACK;
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT date2022;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO date2022;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT name,AVG(weight_kg) FROM animals GROUP BY name;
+SELECT SUM(escape_attempts) as Neutered_Animals FROM animals WHERE neutered = TRUE;
+SELECT SUM(escape_attempts) as not_neutered_animals FROM animals WHERE neutered = FALSE;
+SELECT species,MIN(weight_kg),MAX(weight_kg) FROM animals WHERE species = 'pokemon' GROUP BY species;
+SELECT species,MIN(weight_kg),MAX(weight_kg) FROM animals WHERE species = 'digimon' GROUP BY species;
+SELECT species,AVG(escape_attempts) FROM animals WHERE species = 'pokemon' AND date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+SELECT species,AVG(escape_attempts) FROM animals WHERE species = 'digimon' AND date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
```

Comment:
- [ ] Dear @odaialazzeh, Please add a query to answer Who escapes the most, neutered or not neutered animals? This will be achieved by grouping by `neutered` and taking the avg sum or max.